cmd/imagec/storage.go

```diff
@@ -140,3 +138,39 @@ func WriteImage(image *ImageWithMeta, data io.ReadCloser) error {
 	return nil
 
 }
+
+// WriteMeta will use the existing WriteImage PL function to faciliate a
+// metadata update / overwrite to an existing image
+func WriteMeta(image *ImageWithMeta) error {
+	defer trace.End(trace.Begin(image.ID))
+
+	transport := httptransport.New(options.host, "/", []string{"http"})
+	client := apiclient.New(transport, nil)
+
+	transport.Consumers["application/json"] = httpkit.JSONConsumer()
+	transport.Producers["application/json"] = httpkit.JSONProducer()
+	transport.Consumers["application/octet-stream"] = httpkit.ByteStreamConsumer()
+	transport.Producers["application/octet-stream"] = httpkit.ByteStreamProducer()
+
+	key := new(string)
+	blob := new(string)
+
+	*key = MetaDataKey
+	*blob = image.meta
+
+	r, err := client.Storage.WriteImage(storage.NewWriteImageParams().
+		WithImageID(image.ID).
+		WithParentID(*image.Parent).
+		WithStoreName(image.Store).
+		WithMetadatakey(key).
+		WithMetadataval(blob).
+		WithSum("required"))
```

Comment:
I didn't create a new method on the API -- I'm using the existing WriteImage, so blob sum is required for that API call...but not necessary when only metadata is updated